### ISSUE TITLE
Issue with Schema.Types.ObjectId

### ISSUE THREAD

**Issue by etiennegautier** [tongoose/tongoose#6]
Hello Sparpik,

I tried tongoose, it works pretty well but on schemas with relationships, mongoose does not work...

`export const UserSchema = new Schema({
  address: {
    ref: "Address",
    type: Schema.Types.ObjectId
  },
...`

Output: 
`undefined:4
    type: "Schema".Types."ObjectId"`

**Comment by suiramdev** [comment]
Two years later.
You should've used ObjectId from bson's module:
`import { ObjectId } from "bson";`
Anyway, this module seems to be deprecated 😢

**Comment by kiprasmel** [comment]
> Anyway, this module seems to be deprecated 😢

yeah i have no clue how anyone's using it, i made a hella hacky regex and called it a day, didn't even use it that much myself, and wasn't even good at typescript nor compilers in general:D

i don't think there's much demand so i don't feel any motivation to work on it 😅
